lib/openapi_client_generator.rb

```diff
@@ -206,10 +212,11 @@ def self.at(definition, parameterizer: OpenAPIClientGenerator::Endpoint::Positio
     end
 
     def self.resource_for_path(path)
-      return :unsupported unless path.include? "deployment"
       path_segments = path.split("/").reject{ |segment| segment == "" }
       resource = path_segments[3]
-      resource ||= "repositories"
+
+      supported_resources = ["deployments"]
+      return (supported_resources.include? resource) ? resource : :unsupported
```

Comment:
@tarebyte how does this look? I don't believe `return :unsupported unless path.include?(["deployment"])` is valid 🤔